requirements/pip.txt

```diff
@@ -91,7 +91,8 @@ nltk==3.4.5
 textblob==0.15.3
 
 django-annoying==0.10.5
-django-messages-extends==0.6.0
+# Messages-extends needs a fix for Django 2.2 which isn't released (0.6.1 isn't out yet)
+git+https://github.com/AliLozano/django-messages-extends.git@b1e4f4c#egg=django-messages-extends==0.6.0
```

Comment:
Wonder if because of `0.6.0` this isn't going to get updated, I think we hit something like that in the past

Reply:
When I tried it, it seemed to work correctly.